modules/shared/src/main/scala/org/tessellation/schema/SnapshotOrdinal.scala

```diff
@@ -13,11 +13,15 @@ import eu.timepit.refined.types.numeric.NonNegLong
 import io.circe.{Decoder, Encoder}
 
 @derive(order, show)
-case class SnapshotOrdinal(value: NonNegLong)
+case class SnapshotOrdinal(value: NonNegLong) {
+  def plus(addend: NonNegLong): SnapshotOrdinal = SnapshotOrdinal(value |+| addend)
+}
 
 object SnapshotOrdinal {
   val MinValue: SnapshotOrdinal = SnapshotOrdinal(NonNegLong.MinValue)
 
+  implicit val ordering: Ordering[SnapshotOrdinal] = Ordering.by(_.value.value)
```

Comment:
why `ordering` is required if we already have an `order`?

Reply:
This has been removed.